<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="color">
    <attribute name="title">
     <string>Color</string>
    </attribute>
    <widget class="QSpinBox" name="color_r">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="color_g">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="color_b">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="color_a">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QPushButton" name="color_reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cpu_tab">
    <attribute name="title">
     <string>CPU</string>
    </attribute>
    <widget class="QCheckBox" name="check_cpu_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show CPU name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_cpu_core">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>30</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Show core and clock</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_cpu_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_cpu_core</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_cpu_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_cpu_core</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
